index.rst

```diff
@@ -157,33 +174,24 @@ Merging in work that had started on the old package
-5. Check that the branch in the new clone matches your branch in ``meas_worst``, and if so, continue with the steps below. If the branches still don't match, please ask for help from another developer.
+5. Check that the branch in the new clone matches the branch in ``meas_worst``, and if so, continue with the steps below. If the branches still don't match, please ask for help from another developer.
 
-If the branch in ``meas_worst`` and the branch in the new clone of ``pipe_best`` contain the same set of changes (this will be the case if the other developer had pushed all of their changes to their branch of ``meas_worst`` before you created the fork, or they had followed the above steps if not), the rest of the merging process is simple:
+If the branch in ``meas_worst`` and the branch in the new clone of ``pipe_best`` contain the same set of changes (this will be the case if the other developer had pushed all of their changes to their branch of ``meas_worst`` before you created the fork, or they had followed the above steps if not), the rest of the merging process is straight forward:
 
 1. Rebase the new ``pipe_best`` clone to master: :command:`git rebase master`
 2. Fix any conflicts. There may be a few, if the branch has modified lines near lines that were changed during the rename.
-3. Commit and push the branch to ``pipe_best``, and continue working on your new ``pipe_best`` clone. You can delete your local ``meas_worst`` clone.
+3. Commit and push the branch to ``pipe_best``, and continue working on the new ``pipe_best`` clone. The other developer can delete their local ``meas_worst`` clone.
 
 
 Post-move cleanup
 =================
 
-Because github keeps track of forks, we could not move our renamed package back from `lsst-dm` into `lsst`. Once you are reasonably confident that there is nobody working on ``meas_worst`` and that all relevant work has been moved to ``pipe_best``, you can delete ``meas_worst`` from `lsst` and transfer ``pipe_best`` from `lsst-dm` to `lsst` via the `github transfer repository`_ instructions. Once you've done the transfer, update `repos.yaml` again, removing the ``meas_worst`` entry and changing the ``pipe_best`` entry to refer to the `lsst` organization. This final rename step should not disrupt anything, since github creates a redirect when you use its repository transfer mechanism.
-
-Summary of these steps:
-
-1. Ensure all work has moved to new fork
-2. Delete old package
-3. Move new package to `lsst` organization
-4. Update `lsstsw/etc/repos.yaml`
-
-.. _github transfer repository: https://help.github.com/articles/transferring-a-repository-owned-by-your-organization/
+Once you are reasonably confident that there is nobody working on ``meas_worst`` and that all relevant work has been moved to ``pipe_best``, you can delete ``meas_worst`` from `lsst`, and change your ``pipe_best`` remote to point to its final resting place at ``https://github.com/lsst-dm/pipe_best.git``.
```

Comment:
`meas_worst` should be in `lsst-dm` at this point, and the `pipe_best` remote should be pointed to `lsst`.